### PR TITLE
update server-side-apply state to stable

### LINKED ISSUE #29266: Incorrect feature state for Server-side Apply
According to the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md#server-side-apply-graduates-to-ga): this feature has been graduated to GA in 1.22

The doc still indicates:

`FEATURE STATE: Kubernetes v1.16 [beta]`

```
Disabling the feature

Server Side Apply is a beta feature, so it is enabled by default. To turn this feature gate off, you need to include the --feature-gates ServerSideApply=false flag when starting kube-apiserver. If you have multiple kube-apiserver replicas, all should have the same flag setting.
```

### PR DESCRIPTION
**Pull request by yuswift**
Signed-off-by: yuswift <yuswift2018@gmail.com>

Fixes #29266

### CODE REVIEW

File: content/en/docs/reference/using-api/server-side-apply.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ min-kubernetes-server-version: 1.16
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.16" state="beta" >}}
+{{< feature-state for_k8s_version="v1.22" state="ga" >}}
```
Comment (jihoon-seo) [inline content/en/docs/reference/using-api/server-side-apply.md:15]: Netlify build error:
https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/610d44bd89d3c60007aee9a9
> "alpha, beta, deprecated, stable" is not a valid feature-state, use one of %!q(MISSING)

It seems that `"ga"` should be one of: `"alpha, beta, deprecated, stable"`.

**Review by jihoon-seo — COMMENTED** [review]
@yuswift PTAL!

File: content/en/docs/reference/using-api/server-side-apply.md (hunk 1 above)
Comment (yuswift) [reply content/en/docs/reference/using-api/server-side-apply.md]: Appreciate it, done.

**Review by sftim — COMMENTED** [review]
/lgtm
/approve
/wg api-expression

**Pull request merged**